tst/EnergyPlus/unit/UnitHeater.unit.cc

```diff
@@ -2469,7 +2469,7 @@ TEST_F(EnergyPlusFixture, UnitHeater_SecondPriorityZoneEquipment)
     // check the reheat coil outputis the full capacity
     EXPECT_NEAR(HeatingCoils::HeatingCoil(1).HeatingCoilRate, 17542.3, 1.0);
     // check the unit heater heating coil is handling the remaining load
-    EXPECT_NEAR(HeatingCoils::HeatingCoil(2).HeatingCoilRate, 213.9, 1.0);
+    EXPECT_NEAR(HeatingCoils::HeatingCoil(2).HeatingCoilRate, 203.1, 1.0);
```

Comment:
The HeatingCoilRate for the second coil was changed to fix failed unit test. This result was modified because of change in air distribution unit sensible output as the result of changes in the use *Cp* air calculation.